js/details.js

```diff
@@ -127,9 +106,9 @@
     var target = evt.target;
 
     if (target.dataset.action == 'install') {
-      target.classList.add('disabled');
+      target.disabled = true;
       Details.installTheme().then(() => {
-        target.classList.remove('disabled');
+        target.disabled = false;
```

Comment:
I think we actually want to keep it disabled. (since the theme now has a manifestURL)